modules/mapper-extras/src/main/java/org/opensearch/index/mapper/ScaledFloatFieldMapper.java

```diff
@@ -162,11 +163,21 @@ public ScaledFloatFieldMapper build(BuilderContext context) {
             );
             return new ScaledFloatFieldMapper(name, type, multiFieldsBuilder.build(this, context), copyTo.build(), this);
         }
+
+        @Override
```

Comment:
Lets add a mapping test where scaled float fields are present alongside star tree index mapping as part of both dimensions and metrics